test/e2e/virtual/syncer/storage_pv_test.go

```diff
@@ -0,0 +1,291 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syncer
+
```

Comment:
Could we move those 2 tests in a new `/test/e2e/syncer/storage` package ?

They don't properly test the syncer **virtual workspace** feature, but new controllers added inside the Syncer itself.